composer.json

```diff
@@ -24,15 +24,15 @@
     },
     "require": {
         "ext-simplexml": "*",
-        "php": "^7.1.3",
+        "php": "^7.1.3 || ^8.0",
         "php-http/httplug": "^1.0 || ^2.0",
         "php-http/discovery": "^1.6",
         "php-http/message-factory": "^1.0.2",
         "php-http/client-implementation": "^1.0",
         "psr/simple-cache": "^1.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^7.5",
+        "phpunit/phpunit": "^9.4",
```

Comment:
What about change this line to something like `"phpunit/phpunit": "^7.5|^9.4",` to allow some developpers to still be able to run tests locally with PHP < 8 ?